weblogo/seq_io/clustal_io.py

```diff
@@ -201,7 +207,7 @@ def _scan(fin):
     return
 
 
-def write(fout, seqs):
+def write(fout: TextIO, seqs: List[Seq]) -> None:
```

Comment:
Check, List[Seq] or SeqList?

Reply:
I think `SeqList` ? the other write functions in the other modules use `SeqlList` and  `SeqList` makes sense at this point that you're writing out the SeqList after processing?

Reply:
SeqList to be consistent then.